src/commands/chain/score.ts

```diff
@@ -18,15 +18,15 @@
 import * as Table from 'cli-table3';
 import { HorizontalTable } from 'cli-table3';
 import { command, metadata } from 'clime';
-import { BlockchainScore } from 'symbol-sdk';
+import { ChainInfo } from 'symbol-sdk';
 
 import { ProfileCommand } from '../../interfaces/profile.command';
 import { ProfileOptions } from '../../interfaces/profile.options';
 import { FormatterService } from '../../services/formatter.service';
 
 export class ChainScoreTable {
```

Comment:
> * Finality WS listener channel `finalizedBlock` doesn't seem to have been implemented.
> * Metadata repository has been refactored in rest merged existing endpoints into a single search endpoint. Perhaps, it should have a similar search cmd as mosaic / block
> * Same as above re: search endpoints in Receipt and Restrictions
> * Chain repository endpoints have been merged into 1 `chain/info` now. I think the cli should get refactored to use the same. Also there's new property `latestFinalizedBlock` which should be present in the chain info.
> * Not sure how the transaction search endpoint get implemented before. There are couple of new search criteria (from/to height) have been added in the latest release.
> * Also there are 2 new repositories: SecretLock & HashLock repo. I think it would be better to have  those included in cli

- endpoint finalizedBlock implemented with #5ac616
- Regarding metadata endpoint, the **search** command is more granular as it is implemented in `metadata account`, `metadata mosaic` and `metadata namespace`.
- For receipts and restrictions its a valid point, its not using the search endpoint yet, and a new Issue is available for this in github. Probably out of scope here to get release out.
- Chain repository comments addressed below with #531c3dd
- Transaction search criteria added with #c47a010
- Can you please open new issues for those two new repositories / command needed ? Would be nice to keep track of it. Probably out of scope here for v0.10.x compatibility as it's new features.,

Thank you for your review :+1:

Reply:
Please resolve and/or approve the PR when you see it fits now :+1: